iso8583-tcp-integration-and-message-transformation/iso8583_tcp_inbound/transaction_handler.bal

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2024 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/constraint;
+import ballerina/http;
+import ballerina/lang.array;
+import ballerina/log;
+import ballerina/uuid;
+import ballerinax/financial.iso8583;
+import ballerinax/financial.iso20022.payments_clearing_and_settlement as iso;
+
+configurable BackendServiceConfig backendServiceConfig = ?;
+string backendServiceUrl = backendServiceConfig.host + ":" + backendServiceConfig.port.toString();
+http:Client backendClient = check new (backendServiceUrl);
+
+# Handles inbound transactions and return response.
+#
+# + data - byte array of the incoming message
+# + return - byte array of the response message
+public function handleInbound(byte[] & readonly data) returns byte[]|error {
+
+    string hex = array:toBase16(data);
+    log:printDebug("[Transaction Handler] Received message from the network driver: " + hex);
+    log:printDebug("[Transaction Handler] Recieived data string: " + data.toString());
```

Comment:
Shall we move this to L34?

Reply:
fixed with [add2697](https://github.com/wso2/solutions-financial-demos/pull/1/commits/add2697ad0cbcfc92467395b9f0d49e58ce18ebe)